src/main/java/school/lemon/changerequest/java/Fraction_Number/FractionNumber.java

```diff
@@ -0,0 +1,139 @@
+package school.lemon.changerequest.java.Fraction_Number;
+
+/**
+ * Created by User on 07.12.2016.
+ */
+public class FractionNumber {
+    public FractionNumber() {
+    }
+
+    public FractionNumber(int Numerator, int Denominator) {
+        this.Numerator = Numerator;
+
+        if(Denominator==0){System.out.println("WRONG");}
```

Comment:
`System.out.println("WRONG");` is not the case. Please set denominator to 1 or throw exception.